### PR TITLE
Health lambda

### PR DESCRIPTION
**Pull request by kelvin-dnx**
standardization of alerts

### CODE REVIEW

**Comment by adenot** [comment]
@kelvin-dnx this being a public module, we shouldn't have `dnx` or a specific name format that we use. Please add a variable like alarm_subject_prefix or something like it to use as the subject.

**Comment by kelvin-dnx** [comment]
Hi @adenot, could you check again?

File: health-lambda.py [hunk 1]
```diff
@@ -6,8 +6,17 @@ def lambda_handler(event, context):
...
     event_rule_name = os.environ.get('EVENT_RULE_NAME')
-
-    event_subject = f'ALARM: {event_rule_name}: {event["detail"]["eventTypeCode"]}'
+    alarm_subject_prefix = os.environ.get('ALARM_SUBJECT_PREFIX')
+    
+    # Get customer name from environment variable
+    customer_name = event_rule_name.split('-')[-1]
+    
+    # Format the event rule name with customer name
+    formatted_rule_name = f"dnx-aws-health-event-{event['detail']['eventTypeCode']}-{customer_name}"
```
Comment (adenot) [inline health-lambda.py]: this can be removed?

File: variables.tf [hunk 2]
```diff
@@ -45,3 +45,9 @@ variable "sns_topic_arn" {
   description = "Topic ARN (optional - uses an existing SNS topic)"
   default     = ""
 }
+
+variable "alarm_subject_prefix " {
```
Comment (adenot) [inline variables.tf]: there's a space before the quotes

**Comment by kelvin-dnx** [comment]
@adenot done, thanks for your suggestions

**Review by adenot — APPROVED** [review]

**Pull request merged**